src/AlgebraicGeometry/Schemes/AffineSchemes/Morphisms/Attributes.jl

```diff
@@ -241,20 +245,21 @@ julia> (x1,x2,x3) = gens(R)
 julia> X = subscheme(Y, x1)
 Spectrum
   of quotient
-    of multivariate polynomial ring in 3 variables over QQ
+    of multivariate polynomial ring in 3 variables x1, x2, x3
+      over rational field
     by ideal(x1)
 
 julia> f = inclusion_morphism(X, Y)
 Morphism
-  from [x1, x2, x3]  spec of quotient of multivariate polynomial ring
+  from [x1, x2, x3]  V(x1)
   to   [x1, x2, x3]  affine 3-space over QQ
 given by the pullback function
   x1 -> 0
   x2 -> x2
   x3 -> x3
 
 julia> graph(f)
-(Spec of quotient of multivariate polynomial ring, Morphism: spec of quotient of multivariate polynomial ring -> spec of quotient of multivariate polynomial ring, Morphism: spec of quotient of multivariate polynomial ring -> affine 3-space over QQ with coordinates [x1, x2, x3])
+(V(x1, -x1, x2 - x2, x3 - x3), Morphism: V(x1, -x1, x2 - x2, x3 - x3) -> V(x1), Morphism: V(x1, -x1, x2 - x2, x3 - x3) -> affine 3-space over QQ with coordinates [x1, x2, x3])
```

Comment:
One could choose new names for the codomain in case of an automorphism. Say `x,y,t` and `x',y',t'`

Reply:
Yes this would be a viable solution for this case, but what about morphisms between rings which only happen to have the same variable names by chance? We should discuss this point directly (not on github).